mh/pat.py

```diff
@@ -1781,15 +1815,32 @@ def sendNtcFriendAdd(self, recipient_id, accepted, seq):
         """
         if not accepted:
             return
-        data = b""
-        data += pati.lp2_string(recipient_id)
+
+        # Tell sender their friend request was accepted
+        data_1 = pati.lp2_string(recipient_id)
         friend = pati.FriendData()
-        friend.index = pati.Long(2)
+        friend.index = pati.Long(len(partner_session.get_friends_list()))
+        friend.capcom_id = pati.String(self.session.capcom_id)
+        friend.hunter_name = pati.String(self.session.hunter_name)
+        data_1 += friend.pack()
+        data_1 += struct.pack(">B", accepted)
+        partner_pat_handler = self.server.get_pat_handler(partner_session)
+        if partner_pat_handler is None:
+            return
+        partner_pat_handler.send_packet(PatID4.NtcFriendAdd, data_1, seq)
```

Comment:
`try_send_packet` should be used to avoid a crash if the packet failed to be sent:
```python
handler = self.server.get_pat_handler(partner_session)
if not handler:
    return
handler.try_send_packet(PatID4.NtcFriendAdd, data, seq)
```